GoToPosition:
  frame: "World"
  x: 1.0
  y: 0.0
  z: 0.0

GoToPose:
  frame: "World"
  x: 1.0
  y: -1.0
  z: 0.0
  qx: 0.0
  qy: 0.0
  qz: 0.383
  qw: 0.924

TrackVelocities:
  frame: "World"
  shape: "circle"
  closed: true
  target_tracking_velocity: 1.0
  circle:
    radius: 1.0
    num_points: 3600
  square:
    hight: 2
    num_points: 3600
  spiral:
    start_radius: 0.5
    end_radius: 2.0
    num_loops: 5
    num_points: 7200
  leminscale:
    start_radius: 0.5
    end_radius: 2.0
    num_loops: 5
    num_points: 7200
